File: src/main/java/com/example/domain/bookmark/dto/response/FestivalBookmarkResponse.java
package com.example.domain.bookmark.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class FestivalBookmarkResponse {

    private Long festivalId;
    private Long memberId;
    @JsonProperty("isBookmarked")
    private boolean isBookmarked;
    private Integer bookmarkCount;


}

File: src/main/java/com/example/domain/bookmark/dto/response/FestivalBookmarkResponse.kt
package com.example.domain.bookmark.dto.response

import com.fasterxml.jackson.annotation.JsonProperty

data class FestivalBookmarkResponse(
    val festivalId: Long,
    val memberId: Long,
    @get:JsonProperty("isBookmarked")
    val bookmarked: Boolean,    // ← Boolean을 앞으로
    val bookmarkCount: Int      // ← Int를 뒤로
)